fea-rs/src/parse/grammar/metrics.rs

```diff
@@ -183,14 +248,23 @@ mod tests {
         let (_out, errors, _errstr) = debug_parse_output(fea, |parser| {
             anchor(parser, TokenSet::EMPTY);
         });
-        assert_eq!(errors.len(), 1);
+        assert_eq!(errors.len(), 1, "{_errstr}");
         assert!(
-            errors[0].text().contains("Expected METRIC"),
+            errors[0].text().contains("expected metric"),
             "{}",
             errors[0].text()
         );
     }
 
+    #[test]
+    fn variable_anchor() {
+        let fea = "<anchor 0 (wght=200:-100 wght=900:-150 wght=900,wdth=150:-120)>";
+        let (_out, errors, _errstr) = debug_parse_output(fea, |parser| {
+            anchor(parser, TokenSet::EMPTY);
+        });
+        assert!(errors.is_empty());
+    }
```

Comment:
The volume of changes to eat this, eat that, etc vs the volume of tests is making me skeptical. Are there not more variants to test?

Reply:
I don't write unit tests for this; there's a system of property tests where we take some syntax and then compare the generated parse tree to an expected output. These are in the PR here: https://github.com/cmyr/fea-rs-test-data/pull/21